### PR TITLE
Issues5 - Readme updates

### PR DESCRIPTION
**Pull request by allmightyspiff**
A few updates to the readme, and I have flask just opening config.py instead of using an environment variable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,12 @@ Slick is a reference implementation for using the `SoftLayer Python Bindings <ht
 Installation
 ------------
 **Prerequirements**
-
+Starting with a fresh install of Ubuntu 14.04-64
```
Comment (underscorephil) [inline README.md]: Would be better as an h2

## Starting...

File: README.md [hunk 2]
```diff
@@ -9,9 +9,12 @@ Slick is a reference implementation for using the `SoftLayer Python Bindings <ht
...
 ```bash
 apt-get install libpq-dev sqlite3 python-pip git python-dev
+pip install WTForms  wtforms-html5 WTForms-Components
+pip install flask
+pip install six --upgrade
 ```
 
 **Get everything running**
```
Comment (underscorephil) [inline README.md]: Would be better as an h2

## Get...

File: slick/__init__.py [hunk 3]
```diff
@@ -44,7 +44,7 @@ def add_widget(self, widget):
 
 # Setup the main Flask app
 app = Slick(__name__)
-app.config.from_envvar('SLICK_CONFIG_FILE')
```
Comment (sudorandom) [inline slick/__init__.py]: Why are you removing this? This prevents custom locations for the config location, which is very important when launching a WSGI server using something like gunicorn or uwsgi.

Comment (allmightyspiff) [inline slick/__init__.py]: The config for this project is almost superfluous, and having it be in a set location makes getting it running easier.

I don't think anyone is actually using multiple custom locations, but in case they are, I'll let from_envvar override from_py, but I'll do that in another issue.

File: README.md [hunk 4]
```diff
@@ -8,30 +8,45 @@ Slick is a reference implementation for using the `SoftLayer Python Bindings <ht
...
 
-This will setup the database
+#### Setup the database
 ```bash
 alembic upgrade head   
 ```
 
+
+#### Configuration
+Then you need to setup the config file. Just copy it to config.py, end edit the 2 secret strings. You can switch DB providers, but I would recommend sticking with sqlite for now.
```
Comment (underscorephil) [inline README.md]: ', end edit the' should be ', and edit the?'.  Also, try to reword this to remove the first person language.
